ports/rp2/Makefile

```diff
@@ -6,9 +6,14 @@ BOARD ?= PICO
  
 BUILD ?= build-$(BOARD)
 
+LWIP ?= 0
```

Comment:
I don't think it makes sense to put this into `Makefile`. If a board wants to configure LWIP/WizNet then that should be done in mpconfigboard.cmake.

This is an unfortunate detail of the way the CMake system works, but I don't think adding each individual option to the Makefile is the right way to solve it.